app/Db/TEM_PhaseLock.db

```diff
@@ -349,6 +376,17 @@ record(ai, "$(DEVICE):QI_OFFSET_RBCK") {
   info(autosaveFields, "EGU")
 }
 
+record(calc, "$(DEVICE):QI_OFFSET_RBCK_PS") {
+  field(DESC, "QI Offset, in ps")
+  field(INPA, "$(DEVICE):QI_OFFSET_RBCK CP") # millidegrees
+  field(INPB, "$(DEVICE):QI_GAIN_RBCK CP")   # kHz
+  # ( QI_OFFSET mdeg / 360000 mdeg/cycle) * (1 / QI Gain kHz) * 1000000000 kHz/GHz = QI Offset (ps)  
+  field(CALC, "(A/360000)*(1/B)*1000000000") 
```

Comment:
In case I wasn't the only one curious, calc records define 1/0 as inf, so unchecked division is relatively safe